src/tools/pbs_python.c

```diff
@@ -1648,7 +1648,7 @@ main(int argc, char *argv[], char *envp[])
 		int	print_env = 0;
 		char	*tmp_str = NULL;
 		char	perf_label[MAXBUF];
-		char	perf_action[MAXBUFLEN];
+		char	perf_action[MAXBUFLEN + 13];
```

Comment:
Please add comment saying we need `+ 13`

Reply:
perf_action is used to write hook perf output and description string, where hook perf output size is MAXBUFLEN so need additional size for description string.

Reply:
@birbalsain Can you please add whatever you said in code comment?